base/sysinfo.jl

```diff
@@ -117,7 +117,7 @@ function _cpu_summary(io::IO, cpu::Array{CPUinfo}, i, j)
     println(io)
 end
 
-function cpu_summary(io::IO=STDOUT, cpu::Array{CPUinfo}=cpu_info())
+function cpu_summary(io::IO=STDOUT, cpu::Vector{CPUinfo}=cpu_info())
```

Comment:
It should still be fixed, maybe in a different PR. It makes no sense to allow `Matrix{CPUinfo}` but not `typeof(view(CPUinfo[], 1:0))`.